opencti-platform/opencti-front/src/private/components/data/DataTableToolBar.jsx

```diff
@@ -1692,7 +1692,7 @@ class DataTableToolBar extends Component {
 
           const isManualPromoteSelect = !selectAll
             && selectedTypes.length > 0
-            && selectedTypes.every((type) => promotionTypes.includes(type));
+            && selectedTypes.some((type) => promotionTypes.includes(type));
```

Comment:
I don't understand your fix... so if only some selected entities are Indicators, the button will be available.... we should have all selected entities of type Indicator/observables to have the button